site/docs/4.3/components/popovers.md

```diff
@@ -250,6 +250,25 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
       <td>'scrollParent'</td>
       <td>Overflow constraint boundary of the popover. Accepts the values of <code>'viewport'</code>, <code>'window'</code>, <code>'scrollParent'</code>, or an HTMLElement reference (JavaScript only). For more information refer to Popper.js's <a href="https://popper.js.org/popper-documentation.html#modifiers..preventOverflow.boundariesElement">preventOverflow docs</a>.</td>
     </tr>
+    <tr>
+      <td>sanitizeTemplate</td>
+      <td>boolean</td>
+      <td>true</td>
+      <td>Enable or disable the built-in sanitizer. If activated the <code>'template'</code> option will be sanitized.
+      You can disable this option if you want to sanitize your template with a dedicated library.</td>
```

Comment:
Just disabling this option doesn't actually allow anyone to use a 3rd party library.

Users cannot intercept the code where this value is being used to create DOM content; that is why an option is needed in the first place.

This needs to allow a function to be passed that can be used instead of the built-in method.

Also, I would refrain from naming this `sanitizeTemplate`... perhaps just `sanitize`; this isn't unique to just templates.

It should be required for any plugin that could potentially create/inject insecure markup (e.g. the `content` option in popovers, for example: https://github.com/twbs/bootstrap/blob/v4-dev/js/src/popover.js#L29).

Reply:
It's just for the `template` option so we'll keep that name.

If folks choose to disable that option they have to use a third library.

Reply:
Except that isn't __not__ just for the `template` option....

Reply:
Are you even reading my comments?

Reply:
You're right we should use that for the `content` option if the `html` option is at `true` 👍